iwyu_tool.py

```diff
@@ -26,6 +26,7 @@
 import argparse
 import subprocess
 import re
+from multiprocessing import Pool
```

Comment:
I prefer importing modules and then referring to `module.Type` -- can you change this to `import multiprocessing`?